File: src/main/java/cn/chahuyun/economy/entity/bank/action/Withdraw.java
package cn.chahuyun.economy.entity.bank.action;

import net.mamoe.mirai.contact.User;

/**
 * @author Erzbir
 * @Date: 2022/11/29 21:52
 * 取款功能
 */
public class Withdraw extends BankAction {
    private int money;

    public Withdraw(User user) {
        this.setUser(user);
    }

    @Override
    public void execute() {

    }
}

File: economy-core/src/main/kotlin/cn/chahuyun/economy/model/bank/action/Withdraw.kt
package cn.chahuyun.economy.model.bank.action

import net.mamoe.mirai.contact.User

/**
 * @author Erzbir
 * @Date: 2022/11/29 21:52
 * 取款功能
 */
class Withdraw(user: User, var money: Int = 0) : BankAction(user) {
    override fun execute() {
        // 实现逻辑
    }
}
